FillableFieldworkAIDriver.lua

```diff
@@ -235,3 +235,12 @@ end
 function FillableFieldworkAIDriver:getFillLevelInfoText()
 	return 'NEEDS_REFILLING'
 end
+
+function FillableFieldworkAIDriver:setLightsMask(vehicle)
```

Comment:
What about the UnloadabelFieldworAIDriver? That can be on an unload course as well.